### PR TITLE
Updating front page blurb.

### PR DESCRIPTION
**Pull request by julesfowler**
This addresses #277 .

### CODE REVIEW

**Comment by julesfowler** [comment]
@bourque Any wise words about travis testing would be welcome.

File: exoctk/exoctk_app/templates/index.html [hunk 1]
```diff
@@ -5,7 +5,17 @@
...
+                        Characterization ToolKit (ExoCTK) is an open-source,
+                        modular, data analysis software package focused
+                        primarily on the atmospheric characterization of
+                        exoplanets and time-series observation planning. The
+                        web application runs the most recently released version
+                        of ExoCTK through a web portal designed to provide the
+                        most useful tools for observation planning, forward
+                        modeling, data reduction, limb darkening, light curve
+                        fitting, and retrievals. The full version of ExoCTK is
+                        available through GitHub.  </p>
```
Comment (bourque) [inline exoctk/exoctk_app/templates/index.html]: Looks great, my only comment would be to perhaps add a link to the GitHub repo here?

Comment (julesfowler) [reply exoctk/exoctk_app/templates/index.html]: We have it as a tab on the site. Do you think we need it here too?

Comment (bourque) [reply exoctk/exoctk_app/templates/index.html]: I suppose not!

**Comment by bourque** [comment]
Once we merge #276 we can pull in those changes and hopefully it will solve the Travis issues

File: exoctk/exoctk_app/templates/index.html (hunk 1 above)
Comment (kevin218) [reply exoctk/exoctk_app/templates/index.html]: I like the idea of adding a link to the GitHub repo in this text.  Let's do it!

**Pull request merged**